middle-pgsql.cpp

```diff
@@ -378,8 +367,9 @@ size_t middle_pgsql_t::local_nodes_get_list(nodelist_t &out, osmium::WayNodeList
     size_t wrtidx = 0;
     for (size_t i = 0; i < nds.size(); ++i) {
         if (std::isnan(out[i].lat)) {
-            std::unordered_map<osmid_t, osmNode>::iterator found = pg_nodes.find(nds[i].ref());
-            if(found != pg_nodes.end()) {
+            std::unordered_map<osmid_t, osmNode>::iterator found =
+                pg_nodes.find(nds[i].ref());
+            if (found != pg_nodes.end()) {
```

Comment:
This is part of the code that was just formatted by clang-format. I don't want to touch it at the moment.